src/tests/stub_acl.cc

```diff
@@ -9,10 +9,12 @@
 /* DEBUG: section 28    Access Control */
 
 #include "squid.h"
-#include "acl/Gadgets.h"
 
 #define STUB_API "acl/"
 #include "tests/STUB.h"
 
+#include "acl/Gadgets.h"
 size_t aclParseAclList(ConfigParser &, Acl::Tree **, const char *) STUB_RETVAL(0)
 
+#include "acl/Acl.h"
+const char *AclMatchedName = nullptr;
```

Comment:
Please add this in alphabetical sorting order for the included filepath (eg "acl/A..." above "acl/G...").